detox/test/integration/e2e/flaky.test.js

```diff
@@ -1,4 +1,6 @@
-describe('Stub1', () => {
+const { session } = require('detox/internals');
+
+describe('Flaky', () => {
```

Comment:
It is an important transition. Instead of having multiple `stub*.test.js` with an unclear intent, I divide them into two categories: **flaky** and **passing** tests.

That gives me a firmer ground: the pseudo-flaky test eventually turns 🟢 PASSING, which means that I can test both a happy and an unhappy scenario while expecting the test command (`detox test ...`) to exit with code = 0.

This way the integration tests become stricter and safer.